src/UserService/Controllers/CommunicationController.cs

```diff
@@ -29,14 +29,8 @@ public class CommunicationController : ControllerBase
         {
             OperationResultResponse<Guid> result = command.Execute(request);
```

Comment:
return command.Execute(request). Add status response in command

Reply:
++